### PR TITLE
PLAT-11467 Update BiFields to support default value

### PR DESCRIPTION
**Pull request by symphony-youness**
The goal here is to be able to differentiate 3 things. The element name, Bi properties that should be strings and Bi properties that should be integers.

By setting the default value of the element name and Bi properties that should be strings to null we can make that distinction in the agent. Enabling us to create a default payload with 0 for all fields that would have an Integer as a default value and add the others if present.

### CODE REVIEW

File: src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java [hunk 1]
```diff
@@ -2,87 +2,90 @@
...
-  REQUIRED("required", BiEventType.MESSAGEML_ELEMENT_SENT),
-  MULTI_SELECT("is_multi_select", BiEventType.MESSAGEML_ELEMENT_SENT),
-  MESSAGE_LENGTH("message_length", BiEventType.MESSAGEML_MESSAGE_SENT),
-  ENTITY_JSON_SIZE("entities_json_size", BiEventType.MESSAGEML_MESSAGE_SENT),
-  FREEMARKER("use_freemarker", BiEventType.MESSAGEML_MESSAGE_SENT),
-  POPUPS("popups", BiEventType.MESSAGEML_MESSAGE_SENT),
-  OPENIM("uiactions_openim", BiEventType.MESSAGEML_MESSAGE_SENT),
-  OPENDIALOG("uiactions_opendialog", BiEventType.MESSAGEML_MESSAGE_SENT),
-  COUNT("count", BiEventType.NONE);
+  FORM("form", BiEventType.MESSAGEML_ELEMENT_SENT, null),
```
Comment (symphony-youri) [inline src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java:5]: I don't fully understand how do we decide between null and "0" ?

Comment (symphony-youness) [reply src/main/java/org/symphonyoss/symphony/messageml/bi/BiFields.java:5]: Once in the agent:
- I need to be able to differentiate Integers fields from String fields. We have only 2 string fields in the MESSAGEML_ELEMENT_SENT schema: `style_color` and `type` (those are set to null)
- Then we have the element name (`actionmenu`, `textarea`, `timepicker` ...) when generating a payload for an Element event for Button, we don't want to put by default 'timepicker' to 0. So I set them to null to avoid them being added to the default payload.

In a nutshell, we only want to keep the integers and not have the strings at all. This makes for 3 fields that can be string or null, so it should be okay.

**Review by symphony-youri — APPROVED** [review]
